src/pybind/ceph_argparse.py

```diff
@@ -741,7 +741,7 @@ def matchnum(args, signature, partial=False):
                 else:
                     # it was required, and didn't match, return
                     return matchcnt
-        if desc.req:
+        if desc.numseen > 0:
```

Comment:
With that change, qa/workunits/cephtool/test.sh seems to run up to the point where it tries pg ls-by-pool 0, which fails with Error ENOENT: pool 0 does not exist.  pg ls-by-pool rbd succeeds.  So I guess that's a bug in the test?...